Formula/c/clarinet.rb

```diff
@@ -1,8 +1,8 @@
 class Clarinet < Formula
   desc "Command-line tool and runtime for the Clarity smart contract language"
   homepage "https://www.hiro.so/clarinet"
-  url "https://github.com/hirosystems/clarinet/archive/refs/tags/v2.7.0.tar.gz"
-  sha256 "f3296882f6cc12dab3f28ff73bc551639f5f18eba6065830924ed6f47ca74b8e"
+  url "https://github.com/hirosystems/clarinet/archive/refs/tags/v2.8.0.tar.gz"
+  sha256 "15d4f6c8a8c90371948b4644eb42d61cb31f6a94"
```

Comment:
this is not git sha, but artifact's sha256 checksum

Reply:
Ah, you are so fast!
Sorry, I force pushed just after you.
Thanks 👌

Reply:
np